src/resources/views/learners/base/settings.blade.php

```diff
@@ -48,8 +74,45 @@
 @endsection
 
 @section('body_js')
+    <script src="https://cdn.jsdelivr.net/npm/vue@2.6.12/dist/vue.js"></script>
+    <script>
+        new Vue({
+            el:"#settings",
+
+            data: {
+                firstName: '',
+                lastName: '',
+                email: '',
+                phone: '',
+                errors: [],
+            },
+            methods: {
+                validateForm: function () {
```

Comment:
can we use the vue `vuelidate` component here so we have a general validation library across board

Reply:
if this is not possible, please comment out your reasons in this review